lib/utils/device/logs.ts

```diff
@@ -80,16 +80,19 @@ async function displayDeviceLogs(
 			jsonStream.on('error', (e) => {
 				logger.logWarn(`Error parsing NDJSON log chunk: ${e}`);
 			});
-			logs.once('error', reject);
-			logs.once('end', () => {
+			logs.once('error', handleError);
+			logs.once('end', handleError);
```

Comment:
This change shares a common error handling function for these `error` and `end` events, but I don't think it will make any difference to the case described in the support thread.